features/guests_added.feature

```diff
@@ -11,15 +11,41 @@ Feature: See if you are attending an event
       | Other event       | Look, it changes colors     | 22-Apr-3019 | 422 Treeside Way, Berkeley, CA 94704   | stubhub.com/liquid    | 0        |
     And I am on the events page
 
-  Scenario: I register for an event:
+
+
+  Scenario: I register for an event with no demographics:
     When I follow "rsvp_1"
+    Then I should see "Gender"
     And I fill out the form with the following attributes:
       | firstname    | Test           |
       | lastname     | Guest          |
       | email        | test@guest.com |
     Then I should see "You have successfully registered!"
     And I should see "Test Guest"
 
+  Scenario: Delete a guest
+    When I follow "rsvp_1"
+    And I fill out the form with the following attributes:
+      | firstname    | Test           |
+      | lastname     | Guest          |
+      | email        | test@guest.com |
+    And I am on the events page
+    When I follow "event_1"
+    And I follow "Delete"
+    Then I should see "Guest was successfully removed."
+
+  Scenario: I register for an event with demographics:
+    When I follow "rsvp_1"
+    And I select "Undergrad" from "guest[occupation]"
+    And I select "male" from "guest[gender]"
+    And I select "chemistry" from "guest[department]"
+    And I fill out the form with the following attributes:
```

Comment:
I selected each of these individually just to test the drop down functionality.  i.e. one time use.  For all other guest scenarios you should seed the database using the step that i made in guest steps.